mailpoet/lib/Newsletter/Url.php

```diff
@@ -22,26 +22,26 @@ public function __construct(
   public function getViewInBrowserUrl(
     NewsletterEntity $newsletter = null,
     SubscriberEntity $subscriber = null,
-    $queue = false,
+    SendingQueueEntity $queue = null,
     bool $preview = true
   ) {
-    $data = $this->createUrlDataObject($newsletter, $subscriber, $queue, $preview);
+    $data = $this->createUrlDataObject($preview, $newsletter, $subscriber, $queue);
     return Router::buildRequest(
       ViewInBrowserEndpoint::ENDPOINT,
       ViewInBrowserEndpoint::ACTION_VIEW,
       $data
     );
   }
 
-  public function createUrlDataObject(NewsletterEntity $newsletter = null, SubscriberEntity $subscriber = null, $queue, $preview) {
+  public function createUrlDataObject(
```

Comment:
You had to change the order of parameters because you didn't use nullable types. Also, the `$preview` parameter doesn't have a type here, but we know from the function above that it is of type boolean.